screensaver.turnoff/resources/language/resources.language.en_gb/strings.po

```diff
@@ -0,0 +1,154 @@
+# XBMC Media Center language file
```

Comment:
Can you please follow the string id range?

```
    strings 30000 thru 30999 reserved for plugins and plugin settings
    strings 31000 thru 31999 reserved for skins
    strings 32000 thru 32999 reserved for scripts
    strings 33000 thru 33999 reserved for common strings used in add-ons 
```

See: https://kodi.wiki/view/language_support

Reply:
I am following it, using 33000 thru 33999. Maybe you were looking at an older commit when you wrote this ?